api/lib/controller/transaction/index.js

```diff
@@ -7,21 +7,54 @@ const networks = require('../../../data/networks')
 
 global.fetch = require('node-fetch')
 
-let api
-async function initPolkadotAPI() {
+let kusamaAPI, polkadotAPI, polkadotTestnetAPI
```

Comment:
🤮 
there is no data structure that scales? like an object keyed by the networkId?

Reply:
:sweat_smile:  the vomit emoji